Attachments/adding-attachments.md

```diff
@@ -0,0 +1,251 @@
+## How to upload and attach files from a third party applications
+
+The basic flow of uploading files in Lime Field looks like this:
+
+1. Create a `File` entity
+2. Upload file content to that entity's endpoint
+3. Connect the file entity as attachment to another entity in the API, e.g. `Case`, `Unit`, `Contract`
```

Comment:
We're already talking about working against the API so change this to: `...attachment to another entity, e.g. ...`